packages/pieces/community/mailjet/src/lib/actions/send-email.ts

```diff
@@ -0,0 +1,90 @@
+import { createAction, Property } from '@activepieces/pieces-framework';
+import { AuthenticationType, httpClient, HttpMethod, HttpRequest } from '@activepieces/pieces-common';
+
+export const sendEmail = createAction({
+  name: 'send_email',
+  displayName: 'Send Email',
+  description: 'Send a text, HTML or template email through Mailjet',
+  props: {
+    fromEmail: Property.ShortText({
+      displayName: 'From (Email)',
+      description: 'Sender email, must be verified in Mailjet',
+      required: true
+    }),
+    fromName: Property.ShortText({
+      displayName: 'From (Name)',
+      required: false
+    }),
+    toEmails: Property.Array({
+      displayName: 'Emails of recipients',
+      required: true
+    }),
+    subject: Property.ShortText({
+      displayName: 'Subject',
+      description: undefined,
+      required: true
+    }),
+    textPart: Property.LongText({
+      displayName: 'Text part',
+      description: undefined,
+      required: false
+    }),
+    htmlPart: Property.LongText({
+      displayName: 'HTML part',
+      description: undefined,
+      required: false
+    }),
+    templateId: Property.Number({
+      displayName: 'Template Id',
+      description: 'Template Id (number) defined in Mailjet',
+      required: false
+    }),
+    templateVariables: Property.Object({
+      displayName: 'Template variables',
+      description: undefined,
+      required: false
+    })
+  },
```

Comment:
can you please add piece: mailJetAuth.

Make sure you import it from relative path "../../" not the package name, once you do that 

please remove `as { username: string; password: string; };` it will be no longer needed and type will be inferred from the variable

Reply:
Thanks for explaining!